cypress/integration/logistics/create_new_tour_and_transportation_order_spec.js

```diff
@@ -1,14 +1,15 @@
-import { BPartner } from '../../support/utils/bpartner';
 import { humanReadableNow } from '../../support/utils/utils';
 import { Tour } from '../../support/utils/tour';
+import { Shipper } from '../../support/utils/shipper';
 import { TransportationOrder } from '../../support/utils/transportationOrder';
 
 describe('create new tour and transportation order', function() {
   const date = humanReadableNow();
-  const shipperBPartner = `metasfresh AG`;
+  let shipperBPartner;
   const shipperLocation = `Am Nossbacher Weg 2`;
   const tourName = `TestTour_${date}`;
-  const shipper = `Eigentransport`;
+  const shipperName = `ShipperTest ${date}`;
```

Comment:
Now, that the shipper creation is implemented, I have modified this test as well, in order not to have so many things hardcoded. The downside is that this test is similar(the same) as the one below(create_shipper_set_it_in_transportation_order.js). @metas-ts : should I remove it or should I leave both of them since we have 2 separate tasks?

Reply:
+1 for keeping both